src/net/RTP/RTPSession.cs

```diff
@@ -942,7 +1018,7 @@ private void SetLocalTrackStreamStatus(MediaStreamTrack localTrack, MediaStreamS
         /// be used. IPAddress.Any and IPAddress. Any and IPv6Any are special cases. If they are set the respective
         /// Internet facing IPv4 or IPv6 address will be used.</param>
         /// <returns>A session description payload.</returns>
-        private SDP GetSessionDesciption(List<MediaStreamTrack> tracks, IPAddress connectionAddress)
+        private SDP GetSessionDesciption(List<MediaStreamTrack> tracks, IPAddress connectionAddress, SdpType sdpType)
         {

```

Comment:
Instead of this parameter you can just check if RemoteDescription == null. If the RemoteDescription is not set then it means the offer is being generated. If it it set then the answer is being generated.

Reply:
I'll merge and make this change.